### PR TITLE
Add RapidSim

### PR DESCRIPTION
**Pull request by chrisburr**

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes This is ready for review.

File: recipes/rapidsim/conda-packaging.patch [hunk 1]
```diff
@@ -0,0 +1,294 @@
+From 0c522c5fe2f9e4849d3054b9edad526bb7431385 Mon Sep 17 00:00:00 2001
```
Comment (chrisburr) [inline recipes/rapidsim/conda-packaging.patch]: I'll try to get this patch included in the next upstream release.

File: recipes/rapidsim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"Bs2Jpsiphi\")"
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"Bd2D0rho0\")"
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"Bs2D0Kpi\")"
+    - root -b -q -l "${RAPIDSIM_ROOT}/validation/compareHistograms.C(\"D02Kpi\")"
+
+about:
+  home: https://github.com/gcowan/RapidSim
+  license: MIT
+  license_family: MIT
+  license_file: Licence.md
```
Comment (beckermr) [inline recipes/rapidsim/meta.yaml]: I just want to make sure this is the correct file. Otherwise, LGTM!

Comment (chrisburr) [reply recipes/rapidsim/meta.yaml]: [It is](https://github.com/gcowan/RapidSim/blob/master/Licence.md)

**Comment by chrisburr** [comment]
@beckermr Thanks for the speedy review and merge! 🍰

**Pull request merged**